pages/common/steam.md

```diff
@@ -0,0 +1,24 @@
+# steam
+
+> Video game platform by Valve.
+> More information: <https://developer.valvesoftware.com/wiki/Command_Line_Options#Steam_.28Windows.29>.
+
+- Launch Steam and show debug messages in the terminal:
+
+`steam`
+
+- Launch Steam with a debug console tab included in-app:
+
+`steam -console`
+
+- Log into Steam with the specified credentials:
```

Comment:
Can "Steam" here be replaced with the appropriate URL (e.g. https://<!---->steam.com)?  Unless that's not correct.

Reply:
That would be <https://store.steampowered.com/>.

Reply:
this command lets you skip the login prompt for the steam client gui, not something in your browser. normally you'd launch steam, enter your login info, then be able to access your games. this lets you skip the middle step.

Reply:
Oh, I see.  Then let's leave it as-is. 👍🏻